### DISCUSSION TITLE
Can we create Custom GPT's with OpenWebUI?

### DISCUSSION THREAD

**Discussion by ariesyous** [open-webui/open-webui#2672] [General]
Moving away from ChatGPT Teams and into OpenWebUI, our organization has a number of Custom GPT's we want to port over into OpenWebUI. They are very simple ones, with one or more files in the database and a custom set of instructions which help users get their work done faster. Is there anything like this supported right now in OpenWebUI?

**Comment by tjbck** [comment]
Our plan is to push https://github.com/open-webui/pipelines! (Works with latest dev!)
